baseroms/gc-eu-mq-dbg/file-table-offset.txt

```diff
@@ -0,0 +1 @@
+12F70
```

Comment:
I would say we should just add this to the segments.txt file at the top instead of having two separate txt files. I would also say prefix the number with `0x` just to be clear it is a hex number.

Reply:
It's nice that each line of segments.txt corresponds to a ROM file and I think it would be weird to have an exception to that. We could use a more structured format like json or yaml but I think that's a bit too complicated right now (but maybe it'll be more worth it in the future).

(I added `0x`, Dragorn had the same comment)

Reply:
I don't know. The exception seems worth it just so we don't need to have two different config files (especially when 1 is a single line). Perhaps this is just something we will disagree on.

Reply:
Yeah, I think two config files whose meanings are clear is better than one config file with a strange format

Reply:
I dont like 2 files for this, but if the plan is to use json/yaml/something else soon then I dont really care for now